### ISSUE TITLE
Issue deploying flowable-admin module

### ISSUE THREAD

**Issue by bvedam** [flowable/flowable-engine#1324]
Platform: Windows 10
Flowable version: 6.4.0

Steps to reproduce:
- Deployed flowable-idm after custom configuration and app works fine
- Unzip flowable-admin.war (after renaming ext to zip) and modify flowable-default.properties file
- Zip file and attempted deploying it but it failed with the following error

-----> Downloading Open JDK Like Memory Calculator 3.13.0_RELEASE from https://java-buildpack.cloudfoundry.org/memory-calculator/trusty/x86_64/memory-calculator-3.13.0_RELEASE.tar.gz (found in cache)
error [/tmp/app/WEB-INF/classes/static/bower_components/json3/benchmark/assets/nano.jar]:  missing 12435093 bytes in zipfile
  (attempting to process anyway)
error [/tmp/app/WEB-INF/classes/static/bower_components/json3/benchmark/assets/nano.jar]:  start of central directory not found;
  zipfile corrupt.
  (please check that you have transferred or created the zipfile in the
  appropriate BINARY mode and that you have compiled UnZip properly)
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
note:  /tmp/app/WEB-INF/classes/static/bower_components/json3/vendor/closure-compiler.jar may be a plain executable, not an archive
unzip:  cannot find zipfile directory in one of /tmp/app/WEB-INF/classes/static/bower_components/json3/vendor/closure-compiler.jar or
        /tmp/app/WEB-INF/classes/static/bower_components/json3/vendor/closure-compiler.jar.zip, and cannot find /tmp/app/WEB-INF/classes/static/bower_components/json3/vendor/closure-compiler.jar.ZIP, period.
       Loaded Classes: 17954, Threads: 250
-----> Downloading Client Certificate Mapper 1.8.0_RELEASE from https://java-buildpack.cloudfoundry.org/client-certificate-mapper/client-certificate-mapper-1.8.0_RELEASE.jar (found in cache)
-----> Downloading Container Customizer 2.6.0_RELEASE from https://java-buildpack.cloudfoundry.org/container-customizer/container-customizer-2.6.0_RELEASE.jar (found in cache)
-----> Downloading Container Security Provider 1.16.0_RELEASE from https://java-buildpack.cloudfoundry.org/container-security-provider/container-security-provider-1.16.0_RELEASE.jar (found in cache)
-----> Downloading Spring Auto Reconfiguration 2.5.0_RELEASE from https://java-buildpack.cloudfoundry.org/auto-reconfiguration/auto-reconfiguration-2.5.0_RELEASE.jar (found in cache)
error [/tmp/app/WEB-INF/classes/static/bower_components/json3/benchmark/assets/nano.jar]:  missing 12435093 bytes in zipfile
  (attempting to process anyway)
error [/tmp/app/WEB-INF/classes/static/bower_components/json3/benchmark/assets/nano.jar]:  start of central directory not found;
  zipfile corrupt.
  (please check that you have transferred or created the zipfile in the
  appropriate BINARY mode and that you have compiled UnZip properly)
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
note:  /tmp/app/WEB-INF/classes/static/bower_components/json3/vendor/closure-compiler.jar may be a plain executable, not an archive
unzip:  cannot find zipfile directory in one of /tmp/app/WEB-INF/classes/static/bower_components/json3/vendor/closure-compiler.jar or
        /tmp/app/WEB-INF/classes/static/bower_components/json3/vendor/closure-compiler.jar.zip, and cannot find /tmp/app/WEB-INF/classes/static/bower_components/json3/vendor/closure-compiler.jar.ZIP, period.
error [/tmp/app/WEB-INF/classes/static/bower_components/json3/benchmark/assets/nano.jar]:  missing 12435093 bytes in zipfile
  (attempting to process anyway)
error [/tmp/app/WEB-INF/classes/static/bower_components/json3/benchmark/assets/nano.jar]:  start of central directory not found;
  zipfile corrupt.
  (please check that you have transferred or created the zipfile in the
  appropriate BINARY mode and that you have compiled UnZip properly)
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
note:  /tmp/app/WEB-INF/classes/static/bower_components/json3/vendor/closure-compiler.jar may be a plain executable, not an archive
unzip:  cannot find zipfile directory in one of /tmp/app/WEB-INF/classes/static/bower_components/json3/vendor/closure-compiler.jar or
        /tmp/app/WEB-INF/classes/static/bower_components/json3/vendor/closure-compiler.jar.zip, and cannot find /tmp/app/WEB-INF/classes/static/bower_components/json3/vendor/closure-compiler.jar.ZIP, period.
Uploading droplet, build artifacts cache...
Uploading droplet...
Uploading build artifacts cache...
Exit status 0

**Comment by tijsrademakers** [comment]
Hi,

Can you elaborate a bit more on how you are deploying the Flowable Admin application? I see CloundFoundry messages and Droplet being mentioned in the log, so this doesn't look like a regular Tomcat deployment.

**Comment by vzickner** [comment]
When you are trying to modify a war application this might help:
https://forum.flowable.org/t/start-flowable-flowable-rest-docker-container-with-custom-jars-in-classpath/2654/2

And for a Cloud Foundry deployment this blog post might help:
https://blog.flowable.org/2018/06/11/running-flowable-on-cloud-foundry/

**Comment by bvedam** [comment]
Yes, we deploy to CF and we currently have 6.2.1 in prod, something I forgot to mention. 

And we don't have any custom classes/jars to include. I used WinZip to unzip, modify prop file and zip it back. Thought it was causing the issue as I saw zip file corrupt errors during cf push.

I then looked at the CF logs. Turns out it was due to incorrect port config in the properties file. Appears like the zip file error reported are more like warnings and are not serious errors.

Since this was an issue with our config, I am closing this issue. 

Addl info about the port issue - In 6.2.1 I had blanked the def 8080 port and it defaulted to 443 automatically as our CF apps run on port 443. But with 6.3.1/6.4.0 I had to explicitly say it is 443 for this to work. That was the core issue here.
